src/error-catch-notifier.js

```diff
@@ -0,0 +1,235 @@
+/**
+ * Error catching is enabled
+ * @member {bool}
+ */
+let _isEnabled = false;
```

Comment:
I used accessor functions to get around that but a closure would have been easier. Would you put all of the functions on it, too? Like just export a single object? I'm relying on the singleton behavior pretty heavily so I don't want it to create a new instance on import.